File: backend/src/main/java/com/demo/student_management/dto/diem/DiemCreateRequest.java
package com.demo.student_management.dto.diem;

import lombok.Data;

import java.math.BigDecimal;

@Data
public class DiemCreateRequest {
    private String idHocSinh;
    private String idMonHoc;
    private String idHocKy;
    private BigDecimal diem15;
    private BigDecimal diem45;
    private BigDecimal diemCk;
}

File: frontend/StudentManagement.Shared/Dtos/Diem/DiemCreateRequest.cs
namespace StudentManagement.Shared.Dtos.Diem;

public class DiemCreateRequest
{
    public string IdHocSinh { get; set; } = string.Empty;
    public string IdMonHoc { get; set; } = string.Empty;
    public string IdHocKy { get; set; } = string.Empty;
    public decimal? Diem15 { get; set; }
    public decimal? Diem45 { get; set; }
    public decimal? DiemCk { get; set; }
}
